t/24-worker-engine.t

```diff
@@ -221,17 +222,19 @@ subtest 'problems when syncing tests' => sub {
 };
 
 subtest 'symlink testrepo' => sub {
-    my $worker         = Test::FakeWorker->new;
-    my $client         = Test::FakeClient->new;
-    my $settings       = {DISTRI => 'foo'};
-    my $job            = OpenQA::Worker::Job->new($worker, $client, {id => 12, settings => $settings});
     my $pool_directory = tempdir('poolXXXX');
+    my $worker         = Test::FakeWorker->new(pool_directory => $pool_directory);
+    my $client         = Test::FakeClient->new;
     my $casedir        = testcasedir('foo', undef, undef);
-    $worker->pool_directory($pool_directory);
-    my $result = OpenQA::Worker::Engines::isotovideo::engine_workit($job);
+
+    # error case: CASEDIR missing
```

Comment:
looks like you want to define separate subtests for these? As so far we are indented only one level I think we could have that second level :)